sass/includes/search/_search-filters.scss

```diff
@@ -57,27 +102,27 @@
         &--block {
             display: block;
             max-width: 100%;
+            color: inherit;
+            font-weight: 600;
```

Comment:
Can we use the mixins from `tools/typography` here for `font-weight`?

Reply:
This may be something for discussion on Monday. The version of the `"@nationalarchives/frontend"` package the site is using is `"^0.1.23-prerelease"` and doesn't contain much around font weight tooling. Upgrading the version to `"^0.1.25-prerelease"` introduces a load of breaking changes around missing colours, typography tools and spacings

Reply:
Updated and resolved issues. It wasn't as dramatic as the console suggested. Changes here https://github.com/nationalarchives/ds-ohos-wagtail/pull/10/commits/de1762ac40fcfe9c376bd27a65ffff0fdbbf8298

Reply:
Doing in ETNA now: https://github.com/nationalarchives/ds-wagtail/pull/1466

There are more changes than just the ones needed to compile everything 😉

Reply:
So do you want me to make those changes here, before merging. Or get this merged and make the other changes as a separate task? I don't really have any context to the changes that are made in the commit you've linked to

Reply:
Pulling from upstream later makes most sense here I think. Are you happy for this to be merged in that case?

Reply:
Yup 👍🏼